signer/src/storage/in_memory.rs

```diff
@@ -34,7 +34,7 @@ pub struct Store {
     pub deposit_requests: HashMap<DepositRequestPk, model::DepositRequest>,
 
     /// Deposit requests
-    pub withdraw_requests: HashMap<WithdrawRequestPk, model::WithdrawRequest>,
+    pub withdraw_requests: HashMap<WithdrawRequestPk, model::WithdrawalRequest>,
```

Comment:
Should we change var names while we're at it? i.e. `withdraw_requests` -> `withdrawal_requests`?

Reply:
🤦🏽 oh I missed this, and it was right there.

Reply:
https://github.com/stacks-network/sbtc/pull/528/commits/e0b18d6f08b0bf4d53c60e51a463ba670a964c66